pkg/routes/quickstarts.go

```diff
@@ -17,21 +18,53 @@ func FindQuickstartById(id int) (models.Quickstart, error) {
 	return quickStart, err
 }
 
+func findQuickstartsByName(name string, pagination Pagination) ([]models.Quickstart, error) {
+	var quickStarts []models.Quickstart
+	err := database.DB.Limit(pagination.Limit).Offset(pagination.Offset).Where("name = ?", name).Find(&quickStarts).Error
+	if err != nil {
+		return nil, err
+	}
+
+	return quickStarts, nil
+}
+
 func findQuickstartsByTags(tagTypes []models.TagType, tagValues []string, pagination Pagination) ([]models.Quickstart, error) {
 	var quickstarts []models.Quickstart
 	var tagsArray []models.Tag
-	database.DB.Where("type IN ? AND value IN ?", tagTypes, tagValues).Find(&tagsArray)
-	err := database.DB.Model(&tagsArray).Limit(pagination.Limit).Offset(pagination.Offset).Distinct("id, name, content").Association("Quickstarts").Find(&quickstarts)
+	database.DB.Debug().Where("type IN ? AND value IN ?", tagTypes, tagValues).Find(&tagsArray)
+	err := database.DB.Debug().Model(&tagsArray).Limit(pagination.Limit).Offset(pagination.Offset).Distinct("id, name, content").Association("Quickstarts").Find(&quickstarts)
```

Comment:
I don't think the `Debug` should be enabled when searching for quickstart by tag.

Reply:
I swear I removed them...